### ISSUE TITLE
Channels is now on 3.x

### ISSUE THREAD

**Issue by ShaheedHaque** [CJWorkbench/channels_rabbitmq#29]
With the release of Channels 3.x, and the changes to/coming in pip, I'm seeing this during installs:

```
ERROR: After October 2020 you may experience errors when installing or updating packages. This is because pip will change the way that it resolves dependency conflicts.

We recommend you use --use-feature=2020-resolver to test your packages with the new resolver before it becomes the default.

channels-rabbitmq 1.2.1 requires channels~=2.2, but you'll have channels 3.0.2 which is incompatible.
```

Any chance of an updated release before pip starts breaking would be most welcome :-).

**Comment by adamhooper** [comment]
Thanks for the ping. I'll take a look.

**Comment by adamhooper** [comment]
A status update: since Channels 3.0 is a major version update, I'm going to make the channels_rabbitmq change a major version update, too. And since I don't want to force _two_ major-version updates, I'm being patient.

Since Channels 3.0 is a major version bump, I'm bumping `msgpack`, too. And I'm investigating [aiormq](https://github.com/mosquito/aiormq). `aioamqp` hasn't been released in over a year, and its `pamqp` dependency is obsolete. (Also, its penchant to [hang by default](https://github.com/Polyconseil/aioamqp/issues/149) gives me the willies. Apart from channels_rabbitmq, I find it hard to write clean reconnect code that also works well in unit tests.)

I've submitted https://github.com/mosquito/aiormq/pull/102 to aiormq to support SSL contexts. Once it lands, I'll test it out with `channels_rabbitmq`.

**Comment by ShaheedHaque** [comment]
Sounds like a good plan.

**Comment by adamhooper** [comment]
I got impatient :). I released v1.3.0 today, with Channels 3 support and not much else.
